sdk/digitaltwins/digitaltwins/README.md

```diff
@@ -83,22 +94,59 @@ See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to
         const dependenciesFor = ["testdependenciesFor"];
         const includeModelDefinition = true;
         const maxItemCount = 1;
-        client.digitalTwinModels.list(dependenciesFor, includeModelDefinition, maxItemCount).then((result) => {
-          console.log("The result is:");
-          console.log(result);
-        }).catch((err) => {
-          console.log("An error occurred:");
-          console.error(err);
-        });
+        client.digitalTwinModels
+          .list(dependenciesFor, includeModelDefinition, maxItemCount)
+          .then((result) => {
+            console.log("The result is:");
+            console.log(result);
+          })
+          .catch((err) => {
+            console.log("An error occurred:");
+            console.error(err);
+          });
       });
     </script>
   </head>
   <body></body>
-</html>
+</html
+```
+
+## Key concepts
+
+Azure Digital Twins Preview is an Azure IoT service that creates comprehensive models of the physical environment. It can create spatial intelligence graphs to model the relationships and interactions between people, spaces, and devices.
+You can learn more about Azure Digital Twins by visiting [Azure Digital Twins Documentation](https://docs.microsoft.com/en-us/azure/digital-twins/)
+
+## Examples
+
+Please take a look at the Readme file in
+[samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/digitaltwins/digitaltwins/samples)
+directory for detailed examples on how to use this library.
+
+## Troubleshooting
+
+### Enable logs
+
+You can set the following environment variable to get the debug logging output when using this library.
+
+- Getting debug logs from the Azure Text Analytics client library
+
+```bash
+export AZURE_LOG_LEVEL=verbose
 ```
 
+For more detailed instructions on how to enable logs, you can look at the [@azure/logger package docs](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/logger).
+
+## Next steps
+
+Please take a look at the
+[samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/digitaltwins/digitaltwins/samples)
+directory for detailed examples on how to use this library.
+
+## Contributing
+
+If you'd like to contribute to this library, please read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md) to learn more about how to build and test the code.
+
 ## Related projects
 
 - [Microsoft Azure SDK for Javascript](https://github.com/Azure/azure-sdk-for-js)
-
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js/sdk/README.png)
+  ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js/sdk/README.png)
```

Comment:
I think the URL is wrong here, it's supposed to map to the path of this file, but it's mapping to sdk/README instead

Reply:
Fixed